understanding/22/accessible-authentication-minimum.html

```diff
@@ -141,70 +140,65 @@ <h3>Hiding characters</h3>
-            <li>A web site does not block paste functionality. The user is able to use a third-party password manager to store credentials, copy them, and paste them directly into a login form.</li>
-            <li>A web site uses WebAuthn so the user can authenticate with their device instead of username/password. The user's device could use any available modality. Common methods on laptops and phones are facial-scan, fingerprint, and PIN (Personal Identification Number). The web site is not enforcing any particular use, it is assumed a user will setup a method that suits them.</li>
-            <li>A web site offers the ability to login with a third-party provider using the OAuth method.</li>
-            <li>A web site that requires two-factor authentication allows for multiple options for the 2nd factor, including a USB-based method where the user simply presses a button to enter a time-based token.</li>
-            <li>A web site that requires two-factor authentication displays a QR code which can be scanned by an app on a user's device to confirm identity.</li>
-            <li>A web site that requires two-factor authentication sends a notification to a user's device. The user must use their device's authentication mechanism (for example, user-defined PIN, fingerprint, facial recognition) to confirm identity.</li>
-        </ul>
-
-    </section>
-
-    <section id="techniques">
-        <h2>Techniques for Accessible Authentication</h2>
-
-        <section id="sufficient">
-            <h3>Sufficient Techniques for Accessible Authentication</h3>
-
-            <ol>
-                <li>
-                    <a href="../../techniques/general/G218" class="general">Email link authentication</a>
-                </li>
-                <li>
-                    <a href="../../techniques/html/H100" class="html">Providing properly marked up email and password inputs</a>
-                </li>
-                <li>
-                    Providing WebAuthn as an alternative to username/password (Potential future technique)
-                </li>
-                <li>
-                    Providing a 3rd party login using OAuth (Potential future technique)
-                </li>
-                <li>
-                    Using two techniques to provide 2 factor authentication (Potential future technique)
-                </li>
-            </ol>
-
-        </section>
-
-        <section id="advisory">
-            <h3>Additional Techniques (Advisory) for Accessible Authentication</h3>
-
-        </section>
-
-        <section id="failure">
-            <h3>Failures for Accessible Authentication (Minimum)</h3>
-            <ol>
-               <li>
-                  <a href="../../techniques/failures/F109.html">Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</a>
```

Comment:
Why remove this failure technique link?

Reply:
that was an oversight from the latest resolution to merge conflicts...that's the problem when leaving a PR open for so long that it goes massively out of sync with other changes on main ... makes merge conflict resolution prone to errors like that

Reply:
reintroduced